app/assets/javascripts/common.js

```diff
@@ -6,17 +6,19 @@ showDatePicker: function(){
       });
 },
 partialHideShow: function(){
-    $('#policy_plan_attributes_company_categories_id').on('change',function(){
-      var x= $('#policy_plan_attributes_company_categories_id').val();
-      if (x == 1){
-        $('.vehicle').remove();
-        $('.lic').add();
-      } 
-      else if (x == 2){
-        $('.lic').remove();
-        $('.vehicle').add();
-      }
-    });
+  $('.vehicle').hide();
+  $('.lic').hide();
+  $('#policy_plan_attributes_company_categories_id').on('change',function(){
+    var x= $('#policy_plan_attributes_company_categories_id').val();
+    if (x == 1){
```

Comment:
@GauravMakwana-BTC : Don't use hardcoded values for comparision as already discussed,
can we use switch case instead of if else?